mlflow/tracing/fluent.py

```diff
@@ -576,6 +576,51 @@ def f():
         return None
 
 
+@experimental
+def update_current_trace(
```

Comment:
or we'll add more arguments such as status in the future?

Reply:
> or we chose update_current_trace for future extensibility? For example, we might add new arguments to this function in the future (e.g. status)?

Yea I was thinking of this possibility. Similar API in other tools (e.g. `update_current_trace` in Langfuse) has many more parameters. It might be YAGNI, but the likelihood of new requirements does not sound low for me.